common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -73,6 +73,14 @@ private void Initialize()
 
         private void IgnoreArmCoreDependencyVersions()
         {
+            // Ignore the api-version of subscription operations
```

Comment:
After this change, the uri match fails as:
```
Uri doesn't match:
    request <[https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c?api-version=**>](https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c?api-version=**%3E)
    record  <[https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c?api-version=2021-01-01>](https://management.azure.com/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c?api-version=2021-01-01%3E)
```
We need to find out if this can be wildcard match or we have to wait for all tests migration to finish firstly. We can't change all recording files in 1 PR, too many files for GitHub to handle.